### ISSUE TITLE
Use Contrib Color For User Names on Avatars

### ISSUE THREAD

**Issue by deilann** [HabitRPG/habitica#3735]
For some reason, the hover over names on avatars have a green background. That color green doesn't seem to be used anywhere else. Can it go back to using contributor color?

~~tickets I meant to submit months ago~~

cc: @benmanley, unless someone else wants to take a whack

**Comment by Alys** [comment]
I wouldn't mind having a go at this, and I will unless anyone else says they want it (which they're welcome to do). I also want to make the player's names and levels be invisible until you hover over the avatar, so I'll try that at the same time.

**Comment by benmanley** [comment]
Yes. I agree that this should be changed. Also, the background colour of the level indicator shouldn’t really reflect the contributor level as they’re unrelated.

Maybe there should be a separate colour system of some kind for levels...

Happy for @Alys to take a shot at it, but I’m not sure about making names and levels invisible. (I’ve commented on that [here](https://github.com/HabitRPG/habitrpg/issues/3565#issuecomment-48914491).)

**Comment by Alys** [comment]
It's probably going to be days before I can do this.

For reasons that were mostly stupid, I decided to completely recreate my local vagrant install, and of course it's all gone horribly wrong as it always does. I've decided I'm sick to death of that happening and so I'm trying to work through every error and warning and timeout to find out how to make them go away completely for ever. I won't be doing any dev work until that's finished. 

If someone wants to fix the avatar name colours before I get to it, go right ahead!

EDIT: My local install is working!! I'll get onto this change in the next couple of days.
